e2e/app/page/workspace-edit-page.ts

```diff
@@ -16,7 +16,7 @@ import { makeWorkspaceName } from 'utils/str-utils';
 
 const faker = require('faker/locale/en_US');
 
-export const PageTitle = 'Create Workspace';
+export const PageTitle = 'Create|Duplicate Workspace';
```

Comment:
`waitForLoad()` didn't consider Duplicate Workspace page before. Duplicate and Build page are the same but the finish button are differently labeled.